presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1330,14 +1336,19 @@ private void deleteEmptyStagingDirectories(List<DeclaredIntentionToWrite> declar
 
         private void waitForAsyncRenames()
         {
-            for (CompletableFuture<?> fileRenameFuture : fileRenameFutures) {
-                MoreFutures.getFutureValue(fileRenameFuture, PrestoException.class);
+            ListenableFuture<?> fileRenameFutureAggregate = allAsList(fileRenameFutures);
+            try {
+                MoreFutures.getFutureValue(fileRenameFutureAggregate);
```

Comment:
Any reason why not use `fileRenameFutureAggregate.get()` directly ?

Reply:
`getFutureValue` would throw the original exception, whereas `get` will wrap the exception with `ExecutionException`.

Reply:
It was not static imported before so I thought it is some convention. I'll do that.